tests/nlu/test_spacy_utils.py

```diff
@@ -5,35 +5,13 @@
 from rasa.nlu.utils.spacy_utils import SpacyNLP
 
 
-supported_langauges = [
-    "zh",
-    "da",
-    "nl",
-    "en",
-    "fr",
-    "de",
-    "el",
-    "it",
-    "ja",
-    "lt",
-    "mk",
-    "nb",
-    "pl",
-    "pt",
-    "ro",
-    "ru",
-    "es",
-]
-
-
-@pytest.mark.parametrize("lang", supported_langauges)
-def test_model_fallback_raises_warning(lang: str):
-    """Make sure we raise a warning but we will perform a fallback."""
-    with pytest.warns(FutureWarning):
-        SpacyNLP._check_model_fallback(None, lang, warn=True)
-
-
 def test_model_raises_error_not_exist():
     """It should throw a direct error when a model doesn't exist."""
     with pytest.raises(InvalidModelError):
         SpacyNLP.create({"model": "dinosaurhead"}, RasaNLUModelConfig())
+
+
+def test_model_raises_error_no_model_given():
+    """It should throw a direct error when no model is passed."""
+    with pytest.raises(InvalidModelError):
+        SpacyNLP.create({"model": None}, RasaNLUModelConfig())
```

Comment:
I'm just wondering if it makes more sense to combine these two tests cause they only differ with an argument value for "model".

Reply:
Fair enough. It's two examples that could be one. I usually prefer to go parameterized once I hit `n=3` but I'll make the change.